Doc/library/re.rst

```diff
@@ -637,6 +637,9 @@ form.
       programs that use only a few regular expressions at a time needn't worry
       about compiling regular expressions.
 
+.. class:: RegexFlag
+
+   An :class:`~enum.Enum` class containing the regex options listed below.
```

Comment:
Could you give an example of when you would use this object?

Also, should add a `.. versionadded`. This is a bit of an edge case because the version in which it is being documented (3.11) is different from the one in which it was added (3.6?). I think the versionadded directive should be the version in which it was actually added, with a note that it was first documented in 3.11.

Reply:
There's no particular use case for this object except for static typing, but for that any Enum, any class, can be used as an annotation, so I think it's not appropriate to add a note about such use. (an example of how it's used in typing is in the last message on the BPO from Guido).